src/Stateless/DotGraph/DotGraphFormatter.cs

```diff
@@ -7,78 +7,237 @@ namespace Stateless.DotGraph
     /// <summary>
     /// DOT GraphViz text writer for reflection API.
     /// </summary>
-    public static class DotGraphFormatter
+    public class DotGraphFormatter
     {
+        Dictionary<string, string> clusterDictionary;
```

Comment:
Naming this `clusterDictionary` is probably unnecessary Hungarianization; might be better to go with `_clusters` or similar.

Nit: `_`.